nodes/stampzilla-server/nodeserver.go

```diff
@@ -133,7 +133,7 @@ func (ns *NodeServer) newNodeConnection(connection net.Conn) {
 		case protocol.TypePong:
 			break
 		case protocol.TypePing:
-			writeUpdate(connection, protocol.NewUpdateWithData(protocol.TypePing, nil))
+			serverprotocol.WriteUpdate(connection, protocol.NewUpdateWithData(protocol.TypePing, nil))
```

Comment:
@stamp det här blev snyggt. Här skickar vi till connection och på andra ställen den anropas så är det node man skickar in som första argument :)